authors/apps/profiles/views.py

```diff
@@ -0,0 +1,42 @@
+from rest_framework import status
+from rest_framework.generics import RetrieveAPIView
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.response import Response
+from .exceptions import ProfileDoesNotExist
+from .models import Profile
+from .renderers import ProfileJSONRenderer
+from .serializers import ProfileSerializer, RetrieveProfiles
+
+
+class ProfileRetrieveAPIView(RetrieveAPIView):
+    permission_classes = (AllowAny,)
```

Comment:
Please change this permision class to protect the route

Reply:
I have resolved this